devel/swiftlint/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcodeversion 1.0
+
+github.setup        realm swiftlint 0.52.4
+
+name                swiftlint
+version             0.52.4
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         gmail.com:christos.koninis
+description         Swift linter
+long_description    A tool to enforce Swift style and conventions
+
+checksums           rmd160  cdc5ef3ee9b8804fd45540c31a94f769626deb60 \
+                    sha256  6a76853b566691bf907aac0958464be14f3a063a2b7548bc47840e3f92ed91f2 \
+                    size    828584
+
+# Clearing CPATH because of header conflicts with MacOSX13.sdk when using swiftPM build system
+compiler.cpath
+
+use_configure       no
+use_xcode           yes
+
+build.cmd           /usr/bin/swift
```

Comment:
Choose one of `swift` or `/usr/bin/swift` and ideally use the same command for both `build.cmd` and `test.cmd`?

Reply:
You are right they should both be the same I think plain swift is OK no need for absolut path.